GameData/RP-0/Contracts/Programs.cfg

```
@@ -0,0 +1,679 @@
+	nominalDurationYears = 10
+	baseFunding = 15000000
+
+	REQUIREMENTS
+	{
+		complete_program = CrewedOrbit
+	}
+
+	OBJECTIVES
+	{
+		complete_contract = TODO
+	}
+}
+
+RP0_PROGRAM
+{
+	name = MarsSurfaceExp
+	title = Mars Surface Exploration
+	description = TODO: write up a program description
+	requirementsPrettyText = Complete contracts X and Y
+	objectivesPrettyText = Do this and that
+	nominalDurationYears = 8
+	baseFunding = 4800000
+
+	REQUIREMENTS
+	{
+		complete_program = EarlyInnerPlanetProbes
+	}
+
+	OBJECTIVES
+	{
+		complete_contract = roverMars
+	}
+}
+
+RP0_PROGRAM
+{
+	name = VenusSurfaceExp
+	title = Venus Surface Exploration
+	description = TODO: write up a program description
+	requirementsPrettyText = Complete contracts X and Y
+	objectivesPrettyText = Do this and that
+	nominalDurationYears = 8
+	baseFunding = 4800000
+
+	REQUIREMENTS
+	{
+		complete_program = EarlyInnerPlanetProbes
+	}
+
+	OBJECTIVES
+	{
+		complete_contract = landingVenus
+		minCount = 3
+	}
+}
+
+RP0_PROGRAM
+{
+	name = SmallBodiesFlyby
+	title = Small Bodies Flyby
+	description = TODO: write up a program description
+	requirementsPrettyText = Complete contracts X and Y
+	objectivesPrettyText = Do this and that
+	nominalDurationYears = 10
+	baseFunding = 4000000
+
+	REQUIREMENTS
+	{
+		complete_program = EarlyInnerPlanetProbes
+	}
+
+	OBJECTIVES
+	{
+		ATLEAST
+		{
+			count = 2
+			complete_contract = flybyMercury
+			complete_contract = flybyCeres
+			complete_contract = flybyVesta
+		}
+	}
+}
+
+RP0_PROGRAM
+{
+	name = MercuryExploration
+	title = Mercury Exploration
+	description = TODO: write up a program description
+	requirementsPrettyText = Complete contracts X and Y
+	objectivesPrettyText = Do this and that
+	nominalDurationYears = 8
+	baseFunding = 4800000
+
+	REQUIREMENTS
+	{
+		complete_program = SmallBodiesFlyby
+	}
+
+	OBJECTIVES
+	{
+		complete_contract = landingMercury
+	}
+}
+
+RP0_PROGRAM
+{
+	name = AsteroidExploration
+	title = Asteroid Exploration
+	description = TODO: write up a program description
+	requirementsPrettyText = Complete contracts X and Y
+	objectivesPrettyText = Do this and that
+	nominalDurationYears = 23
+	baseFunding = 8400000
+
+	REQUIREMENTS
+	{
+		complete_program = SmallBodiesFlyby
+	}
+
+	OBJECTIVES //spreadsheet asks for Ceres sample return
+	{
+		complete_contract = orbitCeres
+		complete_contract = orbitVesta
+	}
+}
+
+RP0_PROGRAM
+{
+	name = JupiterObservation
+	title = Jupiter Observation
+	description = TODO: write up a program description
+	requirementsPrettyText = Complete contracts X and Y
+	objectivesPrettyText = Do this and that
+	nominalDurationYears = 8
+	baseFunding = 4000000
+
+	REQUIREMENTS
+	{
+		complete_program = EarlyInnerPlanetProbes
+	}
+
+	OBJECTIVES
+	{
+		ALL
+		{
+			complete_contract = orbitJupiter
+			ATLEAST
+			{
+				count = 2
+				complete_contract = flybyIo
+				complete_contract = flybyEuropa
+				complete_contract = flybyCallisto
+				complete_contract = flybyGanymede
+		}
```

```suggestion
			}
		}
```

There's a missing close brace.

---

Thanks! Committed separately. Cheers for being awesome, as always. <3